<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PIxelPictureRGB</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="l_information">
          <property name="text">
           <string>Photo editor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_selectImage">
          <property name="text">
           <string>Select an image</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="CustomGraphicsView" name="graphicsView"/>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
          <widget class="QLabel" name="l_percentScale">
           <property name="geometry">
            <rect>
             <x>78</x>
             <y>1</y>
             <width>41</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>100%</string>
           </property>
          </widget>
          <widget class="QPushButton" name="b_scalingUp">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>0</y>
             <width>40</width>
             <height>30</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
          <widget class="QPushButton" name="b_downSkaling">
           <property name="geometry">
            <rect>
             <x>1</x>
             <y>1</y>
             <width>40</width>
             <height>30</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="cb_scrolling">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>171</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Display scroll bar</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="b_saveImage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>420</y>
             <width>181</width>
             <height>29</height>
            </rect>
           </property>
           <property name="text">
            <string>Save image</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>82</width>
             <height>28</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>Cursor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pen</string>
            </property>
           </item>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>customgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
